### PR TITLE
feat: support AsyncAPI 2.5.0

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Support AsyncAPI 2.5.0.

**Related issue(s)**
Part of https://github.com/asyncapi/spec/issues/830

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -280,6 +280,7 @@ components:
         - $ref: https://raw.githubusercontent.com/asyncapi/spec-json-schemas/master/schemas/2.2.0.json
         - $ref: https://raw.githubusercontent.com/asyncapi/spec-json-schemas/master/schemas/2.3.0.json
         - $ref: https://raw.githubusercontent.com/asyncapi/spec-json-schemas/master/schemas/2.4.0.json
+        - $ref: https://raw.githubusercontent.com/asyncapi/spec-json-schemas/master/schemas/2.5.0.json
```
Comment (smoya) [inline openapi.yaml:283]: We could just use https://www.asyncapi.com/schema-store/all.schema-store.json and remove the others.

**Review by smoya — CHANGES_REQUESTED** [review]

File: openapi.yaml (hunk 1 above)
Comment (magicmatatjahu) [reply openapi.yaml:283]: Ok, but not in this PR, ok? :)

**Review by smoya — APPROVED** [review]
LGTM! 🚀🌔

**Pull request merged**